### ISSUE TITLE
Add fake config to documentation

### ISSUE THREAD

**Issue by reggi** [mhart/kinesalite#32]
I was trying to figure out a way to run tests without any creds. Seems like I needed this in my file.

`AWS.config.update({accessKeyId: 'FAKE', secretAccessKey: 'FAKE'})`

Could be useful for others.

**Comment by mhart** [comment]
@reggi yeah, I link to the configuration documentation on the README (the "after configuring the SDK" bit)

You can also use any other method mentioned in the sdk config documentation to ensure an access key or secret key, (eg, setting `process.env.AWS_ACCESS_KEY_ID`)

Do you think it should be more explicit, or is the link enough?

**Comment by reggi** [comment]
I was having general trouble getting connected the local Kinesis with `AWS.Kinesis` because of credential issues. Scanned the entire [doc](http://docs.aws.amazon.com/AWSJavaScriptSDK/guide/node-configuring.html), and realized that I really don't even want to have variables connected because I'm testing. It's not super clear that what env is ever being used just given the code. Especially when you don't wanna end up accidentally leave out `{endpoint: 'http://localhost:4567'}` and post to your account. I don't think it's obvious know you don't need creds to use `kinesalite`. Also I think it's safer that these fake creds override your real keys.
